src/dmd/dmodule.d

```diff
@@ -1565,6 +1564,16 @@ extern (C++) final class Module : Package
             buf.prependstring(package_.ident.toChars());
         }
     }
+
+    /** Lazily initializes and returns the escape table.
+    Turns out it eats a lot of memory.
+    */
+    Escape* escapetable()
+    {
+        if (!_escapetable)
+            _escapetable = new Escape;
```

Comment:
Parens.

Reply:
Braces? Looking through the same module, it consistently does not brace single statements (sometimes annoyingly, as in if/else, one with braces and one without). But, when in Rome.

Reply:
OK. When in Rome.